tests/runner.d

```diff
@@ -35,16 +35,17 @@ private int main (string[] args)
         return 1;
     const Args = ["rdmd", ] ~ args[1 .. $] ~
         importPaths.map!(v => "-I" ~ v).array ~
-        lflags.map!(v => "-L" ~ v).array;
+        lflags.map!(v => "-L" ~ v).array ~
+        "-vcolumns";
 
     foreach (test; dirEntries(UnitPath, SpanMode.shallow))
     {
         writeln("Running test on ", test);
         auto pp = pipeProcess(Args ~ test);
         if (pp.pid.wait() != 0)
         {
-            pp.stdout.byLine.each!(a => writeln("[stdout]\t", a));
-            pp.stderr.byLine.each!(a => writeln("[stderr]\t", a));
+            pp.stdout.byLine.each!(writeln);
+            pp.stderr.byLine.each!(line => stderr.writeln(line));
```

Comment:
I can't seem to do `pp.stderr.byLine.each!(stderr.writeln)` here. I get an unusual error:

```D
/Library/D/dmd/src/phobos/std/algorithm/iteration.d(936,48): Error: need `this` for `writeln` of type `@safe void(char[] _param_0)`
```

Reply:
It's not an error message coming from the runner hence it shouldn't go to the runner's `stderr` output.

Reply:
I mean.. the runner itself should just not error at all.

Reply:
I guess we could remove it but having the distinction seemed useful originally.

Reply:
Okay let's just go with this.